packages/ui/components/ui/account-switcher.tsx

```diff
@@ -13,42 +13,85 @@ const MAX_INDEX = 2 ** 32;
 export const AccountSwitcher = ({
   account,
   onChange,
+  filter,
 }: {
   account: number;
   onChange: (account: number) => void;
+  /**
+   * An array of address indexes to switch between, if you want to limit the
+   * options. Must be pre-sorted! `AccountSwitcher` won't sort this for you.
```

Comment:
But it looks like it's being sorted with `[...filter].sort()` no?

Reply:
Whoops, outdated comment. will fix